src/splinterdb.c

```diff
@@ -188,6 +189,15 @@ splinterdb_init_config(const splinterdb_config *kvs_cfg, // IN
 
    shard_log_config_init(&kvs->log_cfg, &kvs->cache_cfg.super, kvs->data_cfg);
 
+   uint64 num_bg_threads[NUM_TASK_TYPES] = {0};
+   num_bg_threads[TASK_TYPE_MEMTABLE]    = kvs_cfg->num_memtable_bg_threads;
+   num_bg_threads[TASK_TYPE_NORMAL]      = kvs_cfg->num_normal_bg_threads;
+   rc                                    = task_system_config_init(
```

Comment:
Minor request: You may want to leave a blank line before this line, so the call to `task_system_config_init()` looks like a function call, and does not fall into the grips of clang-format indentation. (Right now, the code is a bit unreadable.)